pxtpy/pydecompiler.ts

```diff
@@ -32,7 +32,7 @@ namespace pxt.py {
 ///
 /// UTILS
 ///
-const INDENT = "  "
+const INDENT = "\t"
```

Comment:
Maybe just four spaces?

Reply:
I thought about that, but isn't \t more popular these days? vscode's python extension btw uses tabs by default:
![image](https://user-images.githubusercontent.com/6453828/55953310-06107800-5c5c-11e9-8e8e-e49bc2e188e6.png)

Looking at https://stackoverflow.com/questions/119562/tabs-versus-spaces-in-python-programming
you can see there's debate and it does seem clear that the "most correct" way is 4 spaces, but the most popular way answer is tabs.
And personally I'm on the side of tabs for python, even though for most languages I prefer spaces.